src/Sulu/Bundle/AdminBundle/Resources/js/index.js

```diff
@@ -65,3 +68,5 @@ Promise.all([
 ]).then(startApplication);
 
 bundleReady();
+
+export {configPromise};
```

Comment:
I think I would call `bundleReady` after the export... Just to make sure everything has been called before the `bundleReady` call.